templates/pages/target-platforms/boards/up-squared.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Intel Up²
+image: https://assets.ubuntu.com/v1/1f53b707-INTEL_JOULE-LOGO.png?w=200
+description: The Up² board by Intel
+tags: intel, server, lts, desktop, iot
+distributions:
+  - Ubuntu Server 16.04 LTS
+---
+
+# Up&sup2;
+
+## Ubuntu Server
+
+We will walk you through the steps of installing Ubuntu Server on the Intel Up &sup2;. At the end of this process, you will have a fully fledged development or production environment.
+
+### Ubuntu image
+
+* [Up&sup2; - Ubuntu Server 16.04 LTS image](https://up-community.org/media/ms1-release/ubuntu-16.04.3-server-upboard-201017-rc1-amd64.iso)
```

Comment:
Is this a straightforward Ubuntu LTS image or not? If it isn't it would perhaps be helpful to add some information  (e.g. This is an extended version of the standard Ubuntu LTS 16.04.3 which includes...

I only thing that if we are telling people to download 3rd party ISOs, we should make it clear what they are...